demo.env

```diff
@@ -0,0 +1,21 @@
+POSTGRES_PASSWORD=string
+POSTGRES_DB=string
+POSTGRES_USER=string
+POSTGRES_HOST=string
+POSTGRES_PORT=number
+PORT=number
+JWT_SECRET=string
+SESSION_DURATION=number
+EMAIL_VERIFICATION_EXPIRATION_TIME=number
+MOBILE_VERIFICATION_EXPIRATION_TIME=number
+PASSWORD_RESET_EXPIRATION_TIME=number
+EMAIL_HOST=string
+EMAIL_PORT=number
+EMAIL_USER=string
+EMAIL_PASS=string
+HTTPS_PORT=number
+FACEBOOK_APP_ID=string
+FACEBOOK_APP_SECRET=string
+GOOGLE_CLIENT_ID=string
+GOOGLE_CLIENT_SECRET=string
+FIREBASE_DATABASE_URL=string
```

Comment:
Um.. what's this for, man?

Reply:
It's too boring to write full .env file when clone project.